CIP-XXXX/README.md

```diff
@@ -0,0 +1,559 @@
+`BuiltinByteString` result would have to be reversed, which has quadratic cost
+if using only Plutus Core primitives. Thus, we consider these implementations to
+be a good attempt, but not suited to even their intended use, much less more
+general applications.
+
+An alternative possibility for the length argument of
+`builtinIntegerToByteString` would be to treat the argument as either a minimum,
+or a maximum, rather than our more hybrid approach. Specifically, for any input
+`i`, let `d` be the minimum number of digits required to represent `i` as per
+the description of our representation, and let `k` be the length argument.
+Then:
+
+* The _minimum length argument_ approach would produce a result of size 
+  $\min \\{ \texttt{d}, \texttt{k} \\}$; that is, if the length argument is
+  smaller than the minimum required digits, the minimum would be used instead.
+* The _maximum length argument_ approach would produce a result of size `k`,
+  and would error if $\texttt{d} > \texttt{k}$.
+
+Both the minimum length argument, and the maximum length argument, approaches
+have merits. The maximum length argument approach in particular is useful for Case 2:
+in such a setting, we already know the maximum size of
+any element's representation, and if we somehow ended up with a larger
+representation than this, it would be a mistake, which the maximum length
+argument would catch immediately. For the minimum length argument approach, the
+advantage would be more for Case 1: where the length of the
+representation is not known (and the user isn't particularly concerned anyway).
+In such a situation, the user could pass any argument and know that the
+conversion would still work.
+
+However, both of these approaches have disadvantages as well. The minimum
+length argument approach would be more tedious to use with Case 2, as each 
+conversion would require a size check of the resulting
+`BuiltinByteString`. While this is not expensive, it is annoying, and given the
+complexity of the constructions that would be built atop of any finite field
+implementations, it feels unreasonable to require this from users. Likewise, the
+maximum length argument approach is unreasonable for situations like Case 1: the only
+way to establish how many digits would be required involves performing an
+integer logarithm in base 256, which is inefficient and error-prone. Our hybrid
+approach gives the benefits of both the minimum length argument and maximum
+length argument approaches, without the downsides of either: we observe that,
+for situations like Case 1, the length argument would be `0` in practically all
+cases, which is a value that would not be useful in any situation where the maximum 
+length argument approach would be used. This observation allows our approach to work 
+equally well for both Case 1 and 2, with minimal friction.
+
+## Path to Active
+
+### Acceptance Criteria
+
+We consider the following criteria to be essential for acceptance:
```

Comment:
```suggestion
### Acceptance Criteria

- [x] Implemented and released in Plutus V3.

We consider the following criteria to be essential for acceptance:
```
As per https://github.com/cardano-foundation/CIPs/pull/624#issuecomment-2091414191.

Reply:
Are we counting this as "released" when it is on mainnet or on a testnet?

Reply:
Although I was thinking in either case the `Active` status would apply, maybe the wording here should be _included_ (in Plutus V3) if the term "released" here isn't according to common usage.  cc @zliu41 @michaelpj

Reply:
> I was thinking in either case the Active status would apply

Yeah, I think that's right. It has been released in `plutus-core` a number of versions ago, and it has also been included in the latest node pre-release for SanchoNet. It won't be on mainnet until after the Conway HF.

Reply:
@Ryun1 it does sound like the above qualifies it as a "release", since people will already be building on it; let us know if that's OK with you.

Reply:
Sounds good !

Reply:
@kozross (cc @zliu41) if you can admit this language, or equivalent, into the CIP I would be ready to approve it.  I think the addition is important because it demonstrates to the general community (not just Plutus architects) why this CIP is being merged with `Active` status.

Reply:
I added text to this effect to the section immediately following.

Reply:
thanks; I see it now.